### PR TITLE
Add another divider

### PR DESCRIPTION
**Pull request by lucasgsa**
## Description

Added a new divider.

## Additional Information

idk sorry.

### CODE REVIEW

File: area4/dividers.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# area4 dividers list (make sure to only put 12 characters per divider)
+﻿# area4 dividers list (make sure to only put 12 characters per divider)
```
Comment (RDIL) [inline area4/dividers.txt]: What happened here?

Comment (lucasgsa) [reply area4/dividers.txt]: idk why does it appear, but i will review, sorry, i dont change anything in this line.

Comment (lucasgsa) [reply area4/dividers.txt]: i'll send this again

Comment (RDIL) [reply area4/dividers.txt]: fixed

**Comment by RDIL** [comment]
Please fix conflicts and document it

**Pull request closed without merging**